geojson_to_buildingsync.rb

```diff
@@ -1026,15 +1027,30 @@ def convert_feature(feature)
         </auc:Scenario>"
    end
 
-  source += '      </auc:Scenarios>
-        </auc:Report>
+  source += "      </auc:Scenarios>
```

Comment:
This is okay for now, but I think we want to convert this over to REXML::Element.new type of code.

Reply:
Also working on this. At the time I had not discovered rexml.